src/Core/Suite.php

```diff
@@ -111,4 +127,23 @@ protected function runTest(TestInterface $test, TestResult $result)
         $test->setEventEmitter($this->eventEmitter);
         $test->run($result);
     }
+
+    private function focusedTests()
```

Comment:
Yeah, I have no problem with any of that. I tend not to use the "get" prefix in my own stuff, but I know Peridot does tend to use it, so I think I just did it out of habit.

Having a more generic name for the method makes total sense, as does the filter approach. It also becomes more relevant if we change that method from `private` to `protected`, as it could then be interpreted as being part of Peridot's API, so having a future-proof name is a good idea.